docs/mkdocs.yml

```diff
@@ -48,6 +48,7 @@ nav:
         - o365group add: 'cmd/aad/o365group/o365group-add.md'
         - o365group get: 'cmd/aad/o365group/o365group-get.md'
         - o365group list: 'cmd/aad/o365group/o365group-list.md'
+        - o365group conversation list: 'cmd/aad/o365group/o365group-conversation-list.md'
```

Comment:
Please ensure that it is alphabetically sorted. It should go between add and get

Reply:
Sorry about that. I have changed the order.